bundle/manifests/lightspeed-operator-lightspeed-operator-controller-manager_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
why is this PR contributing a new service account definition?

Reply:
ohh, I should have deleted it. it was a residue from a test. Thanks!